src/Test/Perf/infra/fetch_build.csx

```diff
@@ -4,39 +4,130 @@
 #load "../util/tools_util.csx"
 
 using System;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
 using System.Xml;
 
 // TODO: Use actual command line argument parser so we can have help text, etc...
-var branch = Args.Count() == 2 ? Args[0] : "master";
+var sourceFolder = Args[0];
 var destinationFolder = Args.Count() == 2 ? Args[1] : @"C:\Roslyn\Binaries\Release";
```

Comment:
For Automation we drop the binaries into the `Binaries\Release` and as far as we have a mechanism to the install the any binary on-demand we wouldn't need to store the binaries of different drop